mcs/class/corlib/System.Globalization/CultureInfo.cs

```diff
@@ -732,6 +732,9 @@ static void insert_into_shared_tables (CultureInfo c)
 		
 		public static CultureInfo GetCultureInfo (int culture)
 		{
+			if (culture < 1)
```

Comment:
This should be `culture < 0`:
https://github.com/mono/referencesource/blob/master/mscorlib/system/globalization/cultureinfo.cs#L369